### PR TITLE
Justifying container orchestration

### PR DESCRIPTION
**Pull request by softwarepronto**
The original intro to this section read:
When you use .NET Framework console app projects, the option to add Docker support without orchestration isn't supported. You can still use the following procedure, even if you're using only a single Docker project.

There text to guide the reader as to what is orchestration and how it relates to debugging a .NET Framework console app in a local Docker container. The reader is not told that if they add orchestration, it also enables Docker support for the project which allows the project to be debugged in a local Docker container (the title of this help topic).

The other thing that should be reinforced is that this approach applies to .NET Framework console app projects. VS2022 supports adding Docker support to .NET Core console app project for example. Adding docker support by adding support for orchestration applies to a very small number scenarios.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@softwarepronto : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@ghogen

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"

File: docs/containers/edit-and-refresh.md [hunk 1]
```diff
@@ -148,7 +148,11 @@ Often, changes require further inspection. You can use the debugging features of
 
 ## Create a .NET Framework console app
 
-When you use .NET Framework console app projects, the option to add Docker support without orchestration isn't supported. You can still use the following procedure, even if you're using only a single Docker project.
+This section presents how to debug .NET Framework console app project in a local Docker container by first presenting how to add Docker support to the project. It is important to recognize that different project types have different levels of Docker support. There are even different levels of Docker support for .NET Core console app projects versus .NET Framework console app projects.
```
Comment (ghogen) [inline docs/containers/edit-and-refresh.md:151]:
```suggestion
This section presents how to debug a .NET Framework console app project in a local Docker container by first presenting how to add Docker support to the project. It is important to recognize that different project types have different levels of Docker support. There are even different levels of Docker support for .NET Core (including .NET 5 and later) console app projects versus .NET Framework console app projects. 
```

**Review by ghogen — APPROVED** [review]

**Comment by PRMerger15** [comment]
@ghogen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ghogen** [comment]
@softwarepronto 
Thanks, I just added some slight clarifications and grammatical fixes.
#sign-off

**Comment by softwarepronto** [comment]
> @softwarepronto Thanks, I just added some slight clarifications and grammatical fixes. #sign-off

@ghogen thanks... I make a lot of simple changes (backwards compatible to backward compatible). This was my "daring change" but I'm glad someone felt the same way.

**Comment by PRMerger4** [comment]
@jborsecnik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**